resources/views/welcome.blade.php

```diff
@@ -3,19 +3,18 @@
     <x-slot name="header">
         <div class="center center:wide stack stack:lg">
             <h1 itemprop="name">{{ __('The Accessibility Exchange') }}</h1>
-            <x-interpretation name="{{ __('The Accessibility Exchange', [], 'en') }}" />
             <p class="h4">
                 {{ __('Connecting the disability and Deaf communities and their supporters with ') }}<br />{{ __('organizations and businesses to work on accessibility projects together.') }}
             </p>
             @guest
                 <p><a class="cta" href="{{ localized_route('register') }}"> {{ __('Sign up') }}</a></p>
             @endguest
+            <x-interpretation name="{{ __('The Accessibility Exchange', [], 'en') }}" />
```

Comment:
There are two versions of this interpretation, one that includes "Sign up" for guests and one that doesn't for authorized users.